packages/ssi-types/src/types/vc.ts

```diff
@@ -1,6 +1,8 @@
 import { PresentationSubmission } from './pex'
 import { IProofPurpose, IProofType } from './did'
 
+type AdditionalProperties = Record<string, any>
+
 export interface ICredential {
```

Comment:
we might need to add a couple of optional values here:
https://www.w3.org/TR/vc-data-model/#nontransferable-property
credentialStatus
https://www.w3.org/TR/vc-data-model/#dfn-evidence
https://www.w3.org/TR/vc-data-model/#terms-of-use
https://www.w3.org/TR/vc-data-model/#refreshing
https://www.w3.org/TR/vc-data-model/#data-schemas

Reply:
Yeah, will add these later as part of wallet rendering when we encounter them.  Not really in the scope of this PR

Reply:
https://sphereon.atlassian.net/browse/VDX-143